src/Support/Mutators/CastsPropertyMutator.php

```diff
@@ -43,6 +40,20 @@ public function mutate(Model $model, array &$propertyDefinition): void
         }
     }
 
+    /**
+     * @param Model $model
+     * @return array
+     * @throws ReflectionException
+     */
+    private function getCastValues(Model $model): array
+    {
+        $reflectionClass = new \ReflectionClass($model);
+        $castsProperty = $reflectionClass->getProperty('casts');
+        $castsProperty->setAccessible(true);
+
+        return $castsProperty->getValue($model);
```

Comment:
test je nu eigenlijk wel de casts, of alleen de waarden van de casts array? ik begrijp hem nu niet helemaal